app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -105,82 +105,120 @@
+                v-model="form.description"
+                width="w-full"
+                :placeholder="LOREM_IPSOM_TEXT"
+                id="grant-description"
+                :required="true"
+                :rules="isDefined"
+                errorMsg="Please enter a description"
+              />
+            </template>
+          </InputRow>
 
           <!-- Grant website -->
-          <BaseInput
-            v-model="form.website"
-            id="grant-website"
-            label="Grant website"
-            :rules="isValidWebsite"
-            errorMsg="Please enter a valid URL"
-            :required="false"
-          />
+          <InputRow :deleteable="false" :intended="false" text="">
+            <template v-slot:label>Website:</template>
+            <template v-slot:input>
+              <BaseInput
+                v-model="form.website"
+                width="w-full"
+                placeholder="https://"
+                id="grant-website"
+                :rules="isValidWebsite"
+                errorMsg="Please enter a valid URL"
+                :required="false"
+              />
+            </template>
+          </InputRow>
 
           <!-- Grant github -->
-          <BaseInput
-            v-model="form.github"
-            id="grant-github"
-            label="Grant github"
-            :rules="isValidGithub"
-            errorMsg="Please enter a valid Github URL"
-            :required="false"
-          />
+          <InputRow :deleteable="false" :intended="false" text="">
+            <template v-slot:label>Github:</template>
+            <template v-slot:input>
+              <BaseInput
+                v-model="form.github"
+                width="w-full"
+                placeholder="https://"
+                id="grant-github"
+                :rules="isValidGithub"
+                errorMsg="Please enter a valid Github URL"
+                :required="false"
+              />
+            </template>
+          </InputRow>
 
           <!-- Grant twitter handle -->
-          <BaseInput
-            v-model="form.twitter"
-            id="grant-handle"
-            label="Grant twitter"
-            :rules="isValidTwitter"
-            errorMsg="Please enter a valid Twitter handle"
-            :required="false"
-          />
+          <InputRow :deleteable="false" :intended="false" text="">
+            <template v-slot:label>Twitter:</template>
+            <template v-slot:input>
+              <BaseInput
+                v-model="form.twitter"
+                width="w-full"
+                placeholder="@twitterhandle"
+                id="grant-handle"
+                :rules="isValidTwitter"
+                errorMsg="Please enter a valid Twitter handle"
+                :required="false"
+              />
+            </template>
+          </InputRow>
 
           <!-- Submit and cancel buttons -->
-          <div class="flex justify-end pt-6">
+          <div class="px-4 md:px-12 py-12">
```

Comment:
The Cancel and Update Grant buttons are not aligned

![image](https://user-images.githubusercontent.com/17163988/133082597-69a2a237-2c58-4c0c-aa4a-e35676aa1936.png)

Reply:
fixed ! dankee